tests/Get-sbRDSession.Tests.ps1

```diff
@@ -2,14 +2,29 @@
 # Also make sure to keep to the standard Repo Structure e.g.
 <#
 .\<Repo Root>
-    ├───ModuleName
-        ├───Module.psm1
-        ├───Module.psd1
+        ├───ModuleName
+            ├───Module.psm1
+            ├───Module.psd1
     ├───tests
         ├───Get-Example.Tests.ps1
  #>
+
+# UNLESS you have a Vagrant setup with DSC "Examples" folders containing
+# other .psd1 files, in which case, put the modules in a subfolder and
+# adjust the depth of the $ModuleRoot declaration below (\*\*.psd1 for standard,
+# \*\*\*.psd1 for Vagrant/DSC labs)
+<#
+.\<Repo Root>
+    ├───Modules
```

Comment:
Although this will work, its a common standard to use:
```
.\<Repo Root>
    ├───ModuleName	
        ├───Module.psm1
        ├───Module.psd1	
```

I'd look for another solution.
Perhaps flipping this on its head and putting the DSC example files in a sub-folder instead, eg:

```
.\<Repo Root>
    ├───Examples
        ├───RdsLab
            ├───RDSLabConfigData.psd1
            ├───Install-RDSLab.ps1	
```

In future I'll show you how to use BuildHelpers to create Normalized Build Variables: https://github.com/RamblingCookieMonster/BuildHelpers

Reply:
Nice example - will flip.